src/Common/tests/Tests/System/StringTests.cs

```diff
@@ -1019,9 +1019,9 @@ public static void CompareToUnknownComparisonType_StringComparison()
             Assert.Throws<ArgumentException>(() => string.Compare(value, value, (StringComparison)6));
             
             ReadOnlySpan<char> span = value.AsSpan();
-            MemoryTestHelpers.AssertThrows<ArgumentException, char>(span, (_span) => _span.CompareTo(_span, StringComparison.CurrentCulture - 1));
-            MemoryTestHelpers.AssertThrows<ArgumentException, char>(span, (_span) => _span.CompareTo(_span, StringComparison.OrdinalIgnoreCase + 1));
-            MemoryTestHelpers.AssertThrows<ArgumentException, char>(span, (_span) => _span.CompareTo(_span, (StringComparison)6));
+            SpanTestHelpers.AssertThrows<ArgumentException, char>(span, (_span) => _span.CompareTo(_span, StringComparison.CurrentCulture - 1));
```

Comment:
Let's not use "_span" as the name for a local.  Every time I looked at one of these lines I kept initially thinking that _span referred to a field (which makes no sense).  Using an underscore-prefixed name for a local goes against our naming conventions.